keyboards/horrortroll/cat_tkl/info.json

```diff
@@ -0,0 +1,993 @@
+{
+    "keyboard_name": "Cat TKL",
+    "manufacturer": "HorrorTroll",
+    "maintainer": "HorrorTroll",
+    "usb": {
+        "vid": "0x7516",
+        "pid": "0x6003",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP28", "GP27", "GP26", "GP25", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP7", "GP6", "GP11", "GP5"],
+        "rows": ["GP3", "GP2", "GP1", "GP0", "GP10", "GP4"]
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+    "community_layouts": ["tkl_f13_ansi", "tkl_f13_ansi_split_bs_rshift", "tkl_f13_ansi_tsangan", "tkl_f13_ansi_tsangan_split_bs_rshift", "tkl_f13_iso", "tkl_f13_iso_split_bs_rshift", "tkl_f13_iso_tsangan", "tkl_f13_iso_tsangan_split_bs_rshift"],
```

Comment:
Nitpick: 
```suggestion
    "community_layouts": [
        "tkl_f13_ansi", 
        "tkl_f13_ansi_split_bs_rshift", 
        "tkl_f13_ansi_tsangan", 
        "tkl_f13_ansi_tsangan_split_bs_rshift", 
        "tkl_f13_iso", 
        "tkl_f13_iso_split_bs_rshift", 
        "tkl_f13_iso_tsangan", 
        "tkl_f13_iso_tsangan_split_bs_rshift"
    ],
```

Reply:
Thanks, I've just pushed new commit to resolved this.